.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  - pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    env:
+      COMPOSER_HOME: /home/runner/.composer/cache
```

Comment:
This means the cache dir would be `/home/runner/.composer/cache/cache`

Apparently `/github/workspace` is available?
https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#filesystems-on-github-hosted-runners

```suggestion
      COMPOSER_HOME: /github/workspace/composer
```

Reply:
Setting to `/github/workspace/composer` resulted in following error: 

![Screen Shot 2020-08-28 at 15 26 08](https://user-images.githubusercontent.com/5674762/91546998-05f14500-e943-11ea-8fb8-8d1b174a6b16.png)

https://github.com/platformsh/platformsh-cli/pull/977/checks?sha=82828ed167ad6ec9f3004a51973504ffe490bdac